doc/docs.md

```diff
@@ -407,7 +407,7 @@ assert 'aloha!'[0] == `a`
 Both single and double quotes can be used to denote strings. For consistency,
 `vfmt` converts double quotes to single quotes unless the string contains a single quote character.
 
-For raw strings, prepend `r`. Raw strings are not escaped:
+For raw strings, prepend `r`. Raw strings cannot be escaped:
```

Comment:
`Raw strings are not escaped` is stating the defining feature of raw strings.
`cannot be` in this context is less meaningful and even misleading - there is no technical limitation, it is just that you use raw strings, when you do not want the string to have translations/escapes of any kind.

Reply:
I think it would be better as `Character escapes do not work in raw strings:`